doc/manual/channel_interpolation.rst

```diff
@@ -58,7 +58,7 @@ where :math:`G_{ds} \in R^{M \times N}` computes :math:`g_{m}(\boldsymbol{r_i},
 
 To interpolate bad channels, one can simply do:
 
-	>>> evoked.interpolate_bads(reset_bads=False)
+	>>> evoked.interpolate_bads(reset_bads=False)  # doctest: +SKIP
```

Comment:
FYI apparently our doctests aren't really running on CircleCI currently, we'll need to fix it separately though